src/BestBooks.js

```diff
@@ -26,20 +29,52 @@ class BestBooks extends React.Component {
     });
   }
 
+  makeBook = async (newBook) => {
```

Comment:
- A better name for this might be `createBook`, (then we have : `getBooks, createBook, updateBook, deleteBook` for GET, POST, PUT, DELETE --> Create, read, update, delete